slsim/Deflectors/DeflectorTypes/deflector_base.py

```diff
@@ -89,6 +89,16 @@ def mass_ellipticity(self):
         )
         return e1_mass, e2_mass
 
+    @abstractmethod
+    def mass_model_lenstronomy(self, lens_cosmo):
+        """Returns lens model instance and parameters in lenstronomy conventions.
+
+        :param lens_cosmo: lens cosmology model
+        :type lens_cosmo: LensCosmo class
```

Comment:
can you write this as ~lenstronomy.Cosmo.LensCosmo instance?

Reply:
Corrected in the last commit